server/repos.go

```diff
@@ -175,7 +175,12 @@ func PostRepo(c *gin.Context) {
 		httputil.GetURL(c.Request),
 	)
 
-	// TODO(bradrydzewski) verify repo not exists
+	_, err_ := store.Repo(name)
+	if err_ != nil {
```

Comment:
in this case, we want to fail if there is no error (ie `err == nil`) since this means the repository already exists. Also, we should use `err` instead of `err_`